libbeat/publisher/pipeline/pipeline.go

```diff
@@ -68,7 +68,8 @@ type pipelineProcessors struct {
 
 	processors beat.Processor
 
-	disabled bool // disabled is set if outputs have been disabled via CLI
+	disabled   bool // disabled is set if outputs have been disabled via CLI
+	alwaysCopy bool
```

Comment:
Where does this variable get its value?

Reply:
I introduced it for the unit tests enforce the copy of fields. This way, when adding more tests we can see if copy vs. non-copy of fields makes a difference.